DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainerCreateTracksVariadic.h

```diff
@@ -299,12 +299,13 @@ void o2::globaltracking::RecoContainer::createTracksVariadic(T creator) const
     }
     for (unsigned i = 0; i < matchesMCHMID.size(); i++) {
       const auto& match = matchesMCHMID[i];
+      auto [trcTime, isInTF] = match.getTimeMUS(startIR, 256, BCDiffErrCount < MAXBCDiffErrCount);
```

Comment:
why are you using a hard-coded number for the number of orbits and why 256 ? (knowing that the default of the function, 128, seems more in line with the current data taking conditions ?)

Reply:
I use the same hardcoded number as L 133 above to keep the current behavior. If we take this value from somewhere (?) I imagine it should be done consistently within this function.

Reply:
At the moment I used constant 256 orbits as max possible duration of the TF. To be rigorous one should use run specific value (usually 128 but e.g. MC sometimes uses 256) but this would require a guarantee that the GRPECS was loaded, which is not trivial with such a helper method as RecoContainer.